package.json

```diff
@@ -1,12 +1,48 @@
 {
-    "name": "mould",
-    "version": "0.1.0",
-    "private": true,
+    "name": "@mouldjs/mould",
+    "version": "0.1.3",
     "scripts": {
         "dev": "node scripts/ensure-test-workdir && cross-env WORKDIR=./test-mould next dev",
-        "build": "next build",
-        "start": "next start"
+        "clean": "rm -rf release/*",
+        "postbuild": "copyfiles -e \"./node_modules/**/*\" -s \"./**/*\" release && rm -rf release/.mould && rm -rf release/.components && rm -rf release/.next",
+        "prebuild": "yarn clean",
+        "build": "tsc -p tsconfig.package.json",
+        "start": "next start",
+        "preversion": "yarn build",
+        "postversion": "cp package.json release/ && cd release && npm publish"
     },
+    "module": "index.js",
+    "types": "index.d.ts",
+    "files": [
```

Comment:
Do we need to add `.components` to files as well (and therefore remove `rm -rf release/.components` from the `postbuild` script)?
Just to avoid _Runtime Error: Cannot find module './.components'._ on the user side in case users want to import something form `mould` to play with it a bit before running `mould init`

Reply:
Yeah, got it!